test/modules/sdk/objections/service.spec.unit.ts

```diff
@@ -57,6 +57,24 @@ describe("objections SDK service unit tests", () => {
     mockGetBaseAxiosRequestConfig.mockClear();
   });
 
+  it("should return true when getting company eligibility", async () => {
```

Comment:
Could we get a test to cover the false value as well?

Reply:
We could do, but I figured since you're forcing the result anyway so long as it gets the right thing back that would be enough.

Reply:
it should be ok with just true (or false) as there's no different lines of code to execute if it is one value or the other. We're testing that the value returned in the data gets converted to a boolean and returned so that should be enough.

Reply:
actually, you could rename the test to be more generic, "should return boolean when getting company eligibility" I guess with "true" in the title it makes it look like it will only return true, or that there should also be a "false" version.